sdk/metrics-testing/build.gradle.kts

```diff
@@ -1,6 +1,5 @@
 plugins {
   id("otel.java-conventions")
-  id("otel.publish-conventions")
```

Comment:
@jack-berg I think we may have moved packages in this artifact to match the future to help with migration. Because API changes, actually it seems to have had the opposite effect :) With the classes being named the same in old and new, the build currently have issues with collecting coverage, but anyways with the API changes having same named classes with different APIs could cause any amount of weirdness. I think we just need to live with a hard break with this release.

Reply:
Are you planning on merging the rest of the stuff from `metrics-testing` is subsequent PRs?